package/freescale-imx/firmware-imx/firmware-imx.mk

```diff
@@ -4,71 +4,135 @@
 #
 ################################################################################
 
-FIRMWARE_IMX_VERSION = 8.1.1
+FIRMWARE_IMX_VERSION = 8.10
 FIRMWARE_IMX_SITE = $(FREESCALE_IMX_SITE)
 FIRMWARE_IMX_SOURCE = firmware-imx-$(FIRMWARE_IMX_VERSION).bin
 
 FIRMWARE_IMX_LICENSE = NXP Semiconductor Software License Agreement
 FIRMWARE_IMX_LICENSE_FILES = EULA COPYING
 FIRMWARE_IMX_REDISTRIBUTE = NO
 
-FIRMWARE_IMX_BLOBS = sdma vpu
+FIRMWARE_IMX_INSTALL_IMAGES = YES
```

Comment:
```suggestion
FIRMWARE_IMX_INSTALL_IMAGES = YES

ifeq ($(BR2_PACKAGE_LINUX_FIRMWARE),y)
FIRMWARE_IMX_DEPENDENCIES += linux-firmware
endif
```

Reply:
I can do this but was thinking to avoid any custom change if not really required to avoid future maintaianance. I checked ordering in Config.in and firmware-imx takes precedence.

Reply:
> 

Added explicit dependency